src/styles/RepoCard.scss

```diff
@@ -0,0 +1,56 @@
+.card-bg {
```

Comment:
styles files have a naming convention `[fileName].styles.scss`, kindly make sure all style files submitted follow this convention

Reply:
Renamed from `RepoCard.scss` to  `RepoCard.styles.scss`